### ISSUE TITLE
PowerShell 7.1 support

### ISSUE THREAD

**Issue by narfanar** [Pscx/Pscx#72]
Currently Pscx seems to want to clobber a lot of names, so I didn't try to install it. If it's meant to already have support for PS Core 7 and the new versions of the functions are enhanced somehow it may be worth enumerating the benefits to the new versions in the ReadMe.

**Comment by tjmoore** [comment]
Also see #54 #23 . Same commands conflicting as with 5.1, at least in 7.0.3

**Comment by oising** [comment]
Yeah, sorry guys -- we've not been very active on this the last few years. Anywhere you see a clash, it's because we did it first with Pscx. It's Microsoft who keep cloberring us, it's not us clobbering them.

**Comment by rkeithhill** [comment]
@munael Can you give 4.0.0-beta2 a try and tell us how that works WRT clobbering?  https://www.powershellgallery.com/packages/Pscx/4.0.0-beta2

**Comment by moswald** [comment]
@rkeithhill `Install-Module pscx -AllowPrerelease` worked fine for me.

**Comment by BurtHarris** [comment]
Note:  the **-AllowPrerelease** argument does not work on PowerShell 5.1.19041.610

**Comment by Gravifer** [comment]
`Install-Module pscx -AllowPrerelease` does not work for me  (Windows, PowerShell 7.1.1). 
`Get-ShortPath` and `Invoke-NullCoalescing` are now supported by the official team.
It's weird that no attribute has been given to PSCX.

**Comment by BurtHarris** [comment]
I guess you mean no [attribution ](https://www.thefreedictionary.com/attribution#:~:text=n.%201.%20The%20act%20of%20attributing,%20especially%20the,Heritage%C2%AE%20Dictionary%20of%20the%20English%20Language,%20Fifth%20Edition) was given to PSCX.
